examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,805 @@
+       " 'user_item_brands',\n",
+       " 'user_item_intentions',\n",
+       " 'item_category',\n",
+       " 'item_shop',\n",
+       " 'item_brand',\n",
+       " 'TE_user_item_categories_click',\n",
+       " 'TE_user_item_shops_click',\n",
+       " 'TE_user_item_brands_click',\n",
+       " 'TE_user_item_intentions_click',\n",
+       " 'TE_item_category_click',\n",
+       " 'TE_item_shop_click',\n",
+       " 'TE_item_brand_click',\n",
+       " 'TE_user_id_click',\n",
+       " 'TE_item_id_click']"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "schema.column_names"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5bd4e2ee",
+   "metadata": {},
+   "source": [
+    "### Initialize Dataloaders"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9f04211",
+   "metadata": {},
+   "source": [
+    "We're ready to start training, for that, we need to initialize the dataloaders. We'll use Merlin `BatchedDataset` class for reading chunks of parquet files. `BatchedDataset` asynchronously iterate through CSV or Parquet dataframes on GPU by leveraging an NVTabular `Dataset`. To read more about Merlin optimized dataloaders visit [here](https://github.com/NVIDIA-Merlin/models/blob/main/merlin/models/tf/dataset.py#L141)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "4f8b3a89",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python3.8/dist-packages/cudf/core/dataframe.py:1253: UserWarning: The deep parameter is ignored and is only included for pandas compatibility.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "batch_size = 16*1024\n",
+    "LR=0.003\n",
+    "\n",
+    "train_dl = tf_dataloader.BatchedDataset(\n",
+    "    Dataset(output_path + '/train/*.parquet', part_size=\"500MB\"),\n",
+    "    batch_size = batch_size,\n",
+    "    shuffle= True,\n",
+    "    schema = schema,\n",
+    ")\n",
+    "\n",
+    "test_dl = tf_dataloader.BatchedDataset(\n",
+    "    Dataset(output_path + '/test/*.parquet', part_size=\"500MB\"),\n",
+    "    batch_size = batch_size,\n",
+    "    shuffle = False,\n",
+    "    schema = schema,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "748ffde0",
+   "metadata": {},
+   "source": [
+    "### MLP Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95ef3211",
+   "metadata": {},
+   "source": [
+    "<img src=\"./images/mlp.png\"  width=\"30%\">\n",
+    "We will first build a Multi-Layer Percepton (MLP) model. MLP models feed categorical features into embedding layer, concat the embedding outputs and add multiple hidden layers. Note that the `Ali-CCP` dataset has `click` and `conversion` target columns but in this example, we only focus on building different ranking models with binary target column `click`."
```

Comment:
Do we need a linebreak? It seems that there is one missing and the text starts after the image

Reply:
Yes, will fix it. thanks.